python/pip_install/tools/wheel_installer/wheel.py

```diff
@@ -84,16 +84,30 @@ def _as_int(value: Optional[Union[OS, Arch]]) -> int:
     return int(value.value)
 
 
+class MinorVersion(int):
```

Comment:
I'm guessing you're subclassing because you want a type-safe int? This will be confusing because it's easy to lose the type when operations are performed. If you want a type safe int, use NewType instead or manually wrap it in a distinct class (that doesn't inherit from int).

Reply:
I was subclassing because I was not sure if this should be an ABI `enum` or just an `int` and you are right, `int` here works better.